packages/jaeger-ui/src/components/Monitor/ServicesView/operationDetailsTable/__snapshots__/index.test.js.snap

```diff
@@ -7908,15 +7883,11 @@ exports[`<OperationTableDetails> test column render function 1`] = `
                                     width="1em"
                                   >
                                     <path
-                                      d="M512 64C264.6 64 64 264.6 64 512s200.6 448 448 448 448-200.6 448-448S759.4 64 512 64zm0 820c-205.4 0-372-166.6-372-372s166.6-372 372-372 372 166.6 372 372-166.6 372-372 372z"
+                                      d="M512 64C264.6 64 64 264.6 64 512s200.6 448 448 448 448-200.6 448-448S759.4 64 512 64zm32 664c0 4.4-3.6 8-8 8h-48c-4.4 0-8-3.6-8-8V456c0-4.4 3.6-8 8-8h48c4.4 0 8 3.6 8 8v272zm-32-344a48.01 48.01 0 010-96 48.01 48.01 0 010 96z"
```

Comment:
This is the change in SVGs

![image](https://github.com/jaegertracing/jaeger-ui/assets/94157520/7639d87c-62cb-485b-aa60-7cbf1cfb75c7)
(Previous on left, and current on right)

Reply:
@yurishkuro I think we should not move away from Snapshot testing altogether. It would help to catch such changes

Reply:
Thanks that You mentioned will update it.